src/ui/HeaderRenderer.ts

```diff
@@ -328,6 +342,45 @@ export default class HeaderRenderer {
       provider.takeSnapshot(d);
       (<MouseEvent>d3.event).stopPropagation();
     });
+
+    //Heatmap Sort
+    $node.filter((d) => d instanceof MultiValueColumn).append('i').attr('class', 'fa fa-sort').attr('title', 'Sort By').on('click', function (d) {
+      sortDialog(<MultiValueColumn>d, d3.select(this.parentNode.parentNode));
+      (<MouseEvent>d3.event).stopPropagation();
+    });
+    //
+    // $node.filter((d) => d instanceof MultiValueColumn).append('i').attr('class', 'fa fa-sort').attr('title', 'Sort By').on('click', function (d) {
+    //   sortDialogSparkline(<MultiValueColumn>d, d3.select(this.parentNode.parentNode));
+    //   (<MouseEvent>d3.event).stopPropagation();
+    // });
+    //
+    //
+    // $node.filter((d) => d instanceof MultiValueColumn).append('i').attr('class', 'fa fa-sort').attr('title', 'Sort By').on('click', function (d) {
+    //   sortDialogBoxplot(<MultiValueColumn>d, d3.select(this.parentNode.parentNode));
+    //   (<MouseEvent>d3.event).stopPropagation();
+    // });
+    //
+    //
+    // $node.filter((d) => d instanceof MultiValueColumn).append('i').attr('class', 'fa fa-sort').attr('title', 'Sort By').on('click', function (d) {
+    //   sortDialogThresholdBar(<MultiValueColumn>d, d3.select(this.parentNode.parentNode));
+    //   (<MouseEvent>d3.event).stopPropagation();
+    // });
+    //
+    //
+    // $node.filter((d) => d instanceof MultiValueColumn).append('i').attr('class', 'fa fa-sort').attr('title', 'Sort By').on('click', function (d) {
+    //
+    //   sortDialogVerticalBar(<MultiValueColumn>d, d3.select(this.parentNode.parentNode));
+    //   (<MouseEvent>d3.event).stopPropagation();
+    // });
+
+
+    //Renderer Change
+    $node.filter((d) => d instanceof MultiValueColumn || d instanceof NumberColumn).append('i').attr('class', 'fa fa-exchange').attr('title', 'Change Visualization').on('click', function (d) {
```

Comment:
isn't the general condition `getRendererList().length > 1` would ensure extensibility without touching the code